tornado-drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/compiler/plugins/OCLGraphBuilderPlugins.java

```diff
@@ -430,39 +366,6 @@ public boolean apply(GraphBuilderContext b, ResolvedJavaMethod targetMethod, Rec
     }
 
     private static void registerTornadoVMIntrinsicsPlugins(InvocationPlugins plugins) {
-        final InvocationPlugin tprintfPlugin2 = new InvocationPlugin("tprintf", String.class, Object[].class) {
```

Comment:
I see the `tprinf` intrinsic is removed, which makes sense. We can debug directly using the `Debug.printf` method.